packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -76,8 +77,12 @@ export class VaMemorableDate {
   })
   dateBlur: EventEmitter;
 
+  @State() invalidDay: boolean = false;
+  @State() invalidMonth: boolean = false;
+  @State() invalidYear: boolean = false;
```

Comment:
We may eventually want these to be props so that custom validation can take advantage of the `aria-invalid` attribute.

Reply:
Why is this something we would do eventually and not now?

Reply:
I was just trying to limit the scope creep.